### ISSUE TITLE
Document linkarea cards with invisible links

### ISSUE THREAD

**Issue by gfellerph** [swisspost/design-system#6884]
### ✨ Is your feature request related to a problem? Please describe it.

There are teaser cards without visible links, e.g. https://medien.post.ch/de. For these cases, `:focus-within` should be used to display the correct focus ring (2px black border) around the card.

Markup example:

```html
<post-linkarea>
  <div class="card teaser-card">
    <img src="https://picsum.photos/id/38/500/400" alt="My placeholder image" />
    <div>
      <div>
        <h3>Title</h3>
        <p>This is a sample description for the teaser card component.</p>
      </div>
      <a href="#" class="visually-hidden">Link label</a>
    </div>
  </div>
</post-linkarea>
```

### 💡 Describe the solution you'd like

Discuss in pattern discussion what the optimal solution looks like. Possible options include:

1. This teaser card usage is illegal because the real link needs to be visible always
2. We add styles for this use case
3. Ideas?

**Comment by alizedebray** [comment]
Done in https://github.com/swisspost/design-system/pull/7002
